### PR TITLE
Implement Integration test

### LINKED ISSUE #29: Cover the Identity with Integration Tests

### PR DESCRIPTION
**Pull request by kmirik999**
Closes #29

### CODE REVIEW

File: Identity/Identity.IntegrationTests/Identity.IntegrationTests.csproj [hunk 1]
```diff
@@ -4,13 +4,18 @@
         <TargetFramework>net7.0</TargetFramework>
         <ImplicitUsings>enable</ImplicitUsings>
         <Nullable>enable</Nullable>
-
+        <PreserveCompilationContext>true</PreserveCompilationContext>
```
Comment (Bardin08) [inline Identity/Identity.IntegrationTests/Identity.IntegrationTests.csproj:7]: What's that?

File: Identity/Identity.IntegrationTests/IntegrationTest.cs [hunk 2]
```diff
@@ -1,10 +1,69 @@
...
-public class IntegrationTest
+namespace Identity.IntegrationTests
 {
-  [Fact]
-  public void Test1()
-  {
+    public class IntegrationTest
+    {
+        [Fact]
+        public async Task MultipleOperations_IntegrationTest()
```
Comment (Bardin08) [inline Identity/Identity.IntegrationTests/IntegrationTest.cs:12]: Only one test?!

Add more tests & cover all cases, and CORNER-CASES!

**Review by Bardin08 — CHANGES_REQUESTED** [review]
